20_1 TreeList: Adaptability article improvements (#2069)
DevExpress/devextreme-documentation: concepts/05 UI Components/TreeList/10 Columns/50 Adaptability/15 Customize Adaptive Detail Row.md

> Adaptive detail row contains the [Form](/Documentation/ApiReference/UI_Components/dxForm/) UI component. You can implement the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onAdaptiveDetailRowPreparing) handler to customize the Form: change its properties in the **formOptions** attribute of the function's argument. For example, the following code marks the form item whose data field is *"OrderID"* as required:

Reviewer: ```suggestion
Adaptive detail rows contain the [Form](/Documentation/ApiReference/UI_Components/dxForm/) UI component. You can implement the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onAdaptiveDetailRowPreparing) handler to customize the Form: change its properties in the **formOptions** attribute of the function's argument. For example, the following code marks the form item whose data field is set to *"OrderID"* as required:
```

Author: The edited sentence - "For example, the following code marks the form item whose data field is **_set to_** *"OrderID"* as required" - stands for a form object like this:
```
{
  dataFieldName: "OrderID"
}
```
But it should stand for this one:
```
{
  OrderID: someOrderIdValue
}
```
Can the sentence be "For example, the following code marks the form's *"OrderID"* data field as required"?

Reviewer: "marks" doesn't work 
marks it as what?

Author: Marks it as a field that must be filled, e.g.:
![image](https://user-images.githubusercontent.com/66619750/114987696-1f2d0e80-9e9e-11eb-9cb1-b93e4ad4a575.png)

Reviewer: ok, that's fine then

> Adaptive detail row contains the [Form UI component](/Documentation/ApiReference/UI_Components/dxForm/). You can implement the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onAdaptiveDetailRowPreparing') handler to customize it. The following code marks the form item whose data field is *"OrderID"* as required:

Reviewer 2: - "Adaptive detail row contains" - you haven't introduce the adaptive detail row.
- "customize it" - customize what? the row? the Form? the handler?
- What should I change in the handler to customize the Form?

> Adaptive detail row contains the [Form](/Documentation/ApiReference/UI_Components/dxForm/) UI component. You can implement the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onAdaptiveDetailRowPreparing') handler to customize the Form: change its propeties in the **formOptions** attribute of the function's argument. The following code marks the form item whose data field is *"OrderID"* as required:

Reviewer 2: ```suggestion
Adaptive detail row contains the [Form](/Documentation/ApiReference/UI_Components/dxForm/) UI component. You can implement the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onAdaptiveDetailRowPreparing') handler to customize the Form: change its propeties in the **formOptions** attribute of the function's argument. For example, the following code marks the form item whose data field is *"OrderID"* as required:
```

> Adaptive detail row contains the [Form](/Documentation/ApiReference/UI_Components/dxForm/) UI component. You can implement the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onAdaptiveDetailRowPreparing') handler to customize the Form: change its properties in the **formOptions** attribute of the function's argument. For example, the following code marks the form item whose data field is *"OrderID"* as required:

Reviewer 2: ```suggestion
Adaptive detail row contains the [Form](/Documentation/ApiReference/UI_Components/dxForm/) UI component. You can implement the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onAdaptiveDetailRowPreparing) handler to customize the Form: change its properties in the **formOptions** attribute of the function's argument. For example, the following code marks the form item whose data field is *"OrderID"* as required:
```
